Libraries/ART/React-ART.podspec

```diff
@@ -23,7 +23,7 @@ Pod::Spec.new do |s|
   s.homepage               = "http://facebook.github.io/react-native/"
   s.license                = package["license"]
   s.author                 = "Facebook, Inc. and its affiliates"
-  s.platforms              = { :ios => "9.0", :tvos => "9.2", :osx => "10.14" } # TODO(macOS GH#214)
+  s.platforms              = { :ios => "9.0", :tvos => "9.2", :osx => "10.13" } # TODO(macOS GH#214)
```

Comment:
Should this not be 10.14 now Big Sur is out? The normal cadence we did was latest version - 2, which would now be 11.0, 10.15, 10.14

Reply:
Big Sur is [not yet out](https://developer.apple.com/documentation/macos-release-notes), the release date is set to Q4 2020, but I have not selected this version so I'm not able to answer this question (I'm only the messenger here 😉). CC @alloy

Reply:
Ah I've been on the beta so long I just assumed it was out publicly. Yea changing these versions seems fair to me then as we're still supporting 10.13 right now

Reply:
AFAIK Office and Outlook targets 10.13 and there was a discussion even about going down to 10.12.

Reply:
Office targets latest version - 2. When Big Sur is out, react native macOS may want to follow suit and upgrade so we don’t have to have support for OS’s released so long ago. If a client (outlook?) needs 10.12 support for some reason then we should make sure the office suite owners are aware so we can stay consistent

Reply:
I'm not an MS employee so I don't have an internal knowledge, that's was I have heard. Sorry if it's not an accurate information.

Reply:
Ah gotcha, sorry my mistake. Btw thanks for all your contributions to this repo! I think we’re on the same page here with OS support :)

Reply:
I don't have access to the discord, but it seems a bit unnecessary to me to make this change now when we're going to want to undo it in a few weeks when Big Sur hits GM?

Reply:
We want to have this at 10.13, as that’s what a big consumer or RN macOS needs to deploy to.